paparazzi-accessibility/build.gradle

```diff
@@ -0,0 +1,70 @@
+apply plugin: 'kotlin'
+apply plugin: 'org.jetbrains.kotlin.kapt'
+
+// TODO: Remove this duplicate of logic in paparazzi/build.gradle
+def artifactType = Attribute.of('artifactType', String)
+
+configurations {
+  unzip {
+    attributes.attribute(artifactType, ArtifactTypeDefinition.DIRECTORY_TYPE)
+  }
+}
+
+dependencies {
+  registerTransform(org.gradle.api.internal.artifacts.transform.UnzipTransform) {
+    from.attribute(artifactType, ArtifactTypeDefinition.JAR_TYPE)
+    to.attribute(artifactType, ArtifactTypeDefinition.DIRECTORY_TYPE)
+  }
+}
+
+dependencies {
+  api project(':paparazzi')
+
+  testImplementation deps.assertj
+  testImplementation project(':paparazzi')
+
+  // TODO: Remove this duplicate of logic in paparazzi/build.gradle
+  def osName = System.getProperty("os.name").toLowerCase(Locale.US)
+  if (osName.startsWith("mac")) {
+    unzip deps.layoutlib.native.macOsX
+  } else if (osName.startsWith("windows")) {
+    unzip deps.layoutlib.native.windows
+  } else {
+    unzip deps.layoutlib.native.linux
+  }
+}
+
+// TODO: Remove this duplicate of logic in paparazzi/build.gradle
+def generateTestConfig = tasks.register("generateTestConfig") {
+  def resources = "$buildDir/intermediates/paparazzi/resources.txt"
+  outputs.file(resources)
+
+  doLast {
+    File configFile = new File(resources)
+    configFile.withWriter('utf-8') { writer ->
+      writer.writeLine("app.cash.paparazzi")
+      writer.writeLine(".")
+      writer.writeLine("29")
+      writer.writeLine("platforms/android-29/")
+      writer.writeLine(".")
+      writer.writeLine(configurations.unzip.singleFile.path)
+    }
+  }
+}
+
+// TODO: Remove this duplicate of logic in paparazzi/build.gradle
```

Comment:
Is depending on paparazzi module making this script boilerplate necessary?  If so, then we might have to go with the other modular structure after all?

Reply:
Nope this is due to the unit tests I have in the module. I could move them to one of the ones run in the `paparazzi-gradle-plugin` and then we don't need any of the logic since it runs against paparazzi directly via the plugin`

Reply:
So after testing this out it seems I'm going to need to invert the module dependency. I tried using the accessibility repo in the sample module and it was causing issues with the gradle plugin dependencies